src/components/header.js

```diff
@@ -2,25 +2,22 @@ import { Navigation } from "./navigation";
 
 export function Header() {
   return (
-    <>
-      {/* Mobile */}
-      <div className="fixed top-0 left-0 flex w-full shrink-0 items-center justify-between p-10 md:hidden">
+    <header className="absolute left-0 z-10 flex w-full items-start p-10 text-gray-700">
+      <div className="flex flex-1 items-center">
         <Navigation />
+        <span className="ml-10 hidden font-light md:block">
```

Comment:
This is looking good on the page IMO, but it is different than figma - figma seems to show the whole text as clickable and also the whole text is underlined

Reply:
I discussed this one with David and Kate: just underlining Kate's name is preferred